include/jemalloc/internal/emitter.h

```diff
@@ -45,7 +45,9 @@ struct emitter_col_s {
 		int int_val;
 		unsigned unsigned_val;
 		uint32_t uint32_val;
+		uint32_t uint32_t_val;
 		uint64_t uint64_val;
+		uint64_t uint64_t_val;
```

Comment:
Are these two lines used anywhere?

Reply:
Yes, unfortunately they are used inside macros like
    dst->counter_type##_val = rate_per_second(base->counter_type##_val, uptime);
where for the case the provided counter_type is uint64_t not uint64.

If I use the char* union member and cast it to the desired type I get compilation warning or errors at least on MacOS.

I can just rename the union member not to add a new one but this will lead to more places in the code being fixed. Please let me know if you want me to do so.

Reply:
I see. Please leave it as is then.